project-reports/2023/2023-Q4-Hyperledger-Aries.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: 2023 Q4 Hyperledger Aries
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+# Project Health
+
+[Hyperledger Aries](https://www.hyperledger.org/projects/aries) continues at a
+healthy pace, seeing lots of activity in most of the Aries
+sub-communities, including [Aries Framework JavaScript], [Aries Bifold] (mobile
+wallet), [Aries Cloud Agent Python] and [Aries VCX]. Lots of progress being made
+on all of those efforts!
+
+While not a risk to the progress being made in the various open source Aries
+communities, the question of moving some of the Aries projects from the
+[Hyperledger Foundation] to the [Open Wallet Foundation] (OWF) was raised this quarter. A fork
+of a long dormant Aries project (the .NET Framework) has been created in the OWF. A more
+interesting situation is the transfer of the actively maintained [Aries
+Framework JavaScript] repository from Hyperledger to OWF that has been
+[proposed by some Aries Framework JavaScript maintainers]. It's an interesting topic, and
+we'll no doubt see more discussion and action related to Aries and the OWF in coming months.
```

Comment:
This is definitely a topic the TOC should discuss to develop first an understanding of the dynamic at play and then a position as to where we stand on this issue.

Reply:
+1